src/mappings/util.ts

```diff
@@ -147,6 +148,13 @@ export const hexToString = (input: string): string => {
   return removeNullChars(str);
 };
 
+export const maybeHexToString = (input: string): string => {
```

Comment:
```suggestion
/**
* If given string that begins with "0x", this method will create a string from its binary representation. Otherwise it returns the string as it is.

@example "0x1230" -> SomeTicker
*/
export const coerceHexToString = (input: string): string => {
```

I see why we need this method, but at first I thought it was weird.